GVFS/GVFS.UnitTests/Prefetch/DiffHelperTests.cs

```diff
@@ -11,9 +11,27 @@
 
 namespace GVFS.UnitTests.Prefetch
 {
-    [TestFixture]
+    [TestFixtureSource(TestRunners)]
     public class DiffHelperTests
-    {
+    {
+        public const string TestRunners = "Runners";
+
+        private static readonly bool[] SymLinkSupport = new bool[]
+        {
+            true,
+            false,
+        };
+
+        public DiffHelperTests(bool symLinkSupport)
+        {
+            DiffHelper.SetIncludeSymLinks(symLinkSupport);

```

Comment:
I can't see how the test is validating any difference in behavior based on `symLinkSupport`.  Can you add a test that validates the changes to `ShouldIncludeResult`? (e.g verifying that `FileAddOperations` does\doesn't contain the symlinks)